pkg/landscaper/installations/executions/template/gotemplate.go

```diff
@@ -480,3 +483,62 @@ func resolveComponents(defaultCD *cdv2.ComponentDescriptor, list *cdv2.Component
 
 	return components, nil
 }
+
+func getImageVectorGoFunc(cd *cdv2.ComponentDescriptor, list *cdv2.ComponentDescriptorList) func(args ...interface{}) map[string]interface{} {
+	return func(args ...interface{}) map[string]interface{} {
+		internalCd := cd
+		internalComponents := list
+
+		if len(args) > 2 {
+			panic("Too many arguments for getImageVectorGoFunc")
```

Comment:
```suggestion
			panic("Too many arguments for generateImageOverwrite")
```
this should be the user facing function not the go function